Doc/library/winreg.rst

```diff
@@ -131,6 +131,9 @@ This module offers the following functions:
 
    *This method can not delete keys with subkeys.*
 
+   The key identified by the *key* parameter must have been opened with
+   :const:`KEY_SET_VALUE` access.
+
```

Comment:
Per the message I left on the tracker, this should state that `key` can have any access. It's only used to resolve the relative path in `sub_key`.